src/components/Icon.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <component class="nq-icon" :is="componentName"/>
+</template>
+
+<script lang="ts">
+import {Component, Prop, Vue} from 'vue-property-decorator';
+
+import AlertTriangleIcon from '@nimiq/style/src/icons/alert-triangle.svg';
```

Comment:
A comment here that these svgs are imported as components via vue-sv-loader would be nice.